openverse_catalog/dags/data_refresh/data_refresh_types.py

```diff
@@ -17,28 +17,28 @@ class DataRefresh:
 
     Required Constructor Arguments:
 
-    media_type: string describing the media type to be refreshed.
+    media_type: str describing the media type to be refreshed.
 
     Optional Constructor Arguments:
 
-    default_args:      dictionary which is passed to the airflow.dag.DAG
-                       __init__ method.
-    start_date:        datetime.datetime giving the
-                       first valid execution_date of the DAG.
-    schedule_interval: string giving the schedule on which the DAG should
-                       be run.  Passed to the airflow.dag.DAG __init__
-                       method.
-    execution_timeout: datetime.timedelta giving the amount of time a given data
-                       pull may take.
-    doc_md:            string which should be used for the DAG's documentation markdown
+    default_args:         dictionary which is passed to the airflow.dag.DAG
+                          __init__ method.
+    start_date:           datetime.datetime giving the
+                          first valid execution_date of the DAG.
+    schedule_interval:    string giving the schedule on which the DAG should
+                          be run.  Passed to the airflow.dag.DAG __init__
+                          method.
+    data_refresh_timeout: int giving the amount of time in seconds a given
+                          data pull may take.
+    doc_md:               string used for the DAG's documentation markdown
```

Comment:
```suggestion
    doc_md:               str used for the DAG's documentation markdown
```

To be consistent with the change to the media_type docstring above?

Reply:
Oops, definitely! Good catch